internal/internal_workflow_client.go

```diff
@@ -75,7 +75,10 @@ const (
 	pollUpdateTimeout = 60 * time.Second
 )
 
-var maxListArchivedWorkflowTimeout = time.Minute * 3
+var (
+	maxListArchivedWorkflowTimeout = time.Minute * 3
+	errInvalidWorkflowOperation    = fmt.Errorf("invalid WorkflowOperation")
+)
```

Comment:
These should probably be consts

Reply:
Right; I can make `maxListArchivedWorkflowTimeout` a const, but calling `fmt.Errorf` in Go is not allowed for a const.